### PR TITLE
pull DB operator and DB image from OCIR

### PR DESCRIPTION
**Pull request by sankarpn**
We were pulling db operator and db images from ocr. Now this pr changes it to pull those images from ocir.

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9378/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9376/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java [hunk 1]
```diff
@@ -62,14 +62,12 @@
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.concurrent.TimeUnit.SECONDS;
 import static oracle.weblogic.kubernetes.TestConstants.BASE_IMAGES_REPO_SECRET;
-import static oracle.weblogic.kubernetes.TestConstants.DB_OPERATOR_YAML_URL;
+import static oracle.weblogic.kubernetes.TestConstants.DB_IMAGE_NAME;
+import static oracle.weblogic.kubernetes.TestConstants.DB_OPERATOR_IMAGE;
+import static oracle.weblogic.kubernetes.TestConstants.OCIR_SECRET_NAME;
 import static oracle.weblogic.kubernetes.TestConstants.OCR_DB_19C_IMAGE_TAG;
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java:68]: Rename it to OCIR_DB_19C_IMAGE_TAG  or simple DB_19C_IMAGE_TAG

**Review by anpanigr — COMMENTED** [review]
Add the jenkin run result to the description

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java (hunk 1 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java:68]: We have other images with OCR prefix.

  public static final String OCR_WEBLOGIC_IMAGE_NAME = "middleware/weblogic";
  public static final String OCR_WEBLOGIC_IMAGE_TAG = "12.2.1.4";
  public static final String OCR_FMWINFRA_IMAGE_NAME = "middleware/fmw-infrastructure";
  public static final String OCR_FMWINFRA_IMAGE_TAG = "12.2.1.4";
  public static final String OCR_DB_IMAGE_NAME = "database/enterprise";
  public static final String OCR_DB_IMAGE_TAG = "12.2.0.1-slim";
  public static final String OCR_DB_19C_IMAGE_TAG = "19.3.0.0";

Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java:68]: We will remove them when we will consolidate the environment variables.

**Review by anpanigr — APPROVED** [review]
LGTM.  
Backport the change to release/3.3 and release/3.4

**Pull request merged**